### PR TITLE
Added mimeType lower case comparison

### PR DESCRIPTION
**Pull request by index23**
Additional lower case comparison of mime type to commit 85ba5e4

### CODE REVIEW

**Comment by mimse** [comment]
Hi @index23 

I did something simllear to the dash parserv

File: lib/hls/hls_parser.js [hunk 1]
```diff
@@ -1640,7 +1640,7 @@ shaka.hls.HlsParser = class {
           segmentResponse.data, initSegmentResponse.data);
     }
 
-    if (mimeType == 'video/mp2t') {
+    if (mimeType && mimeType.toLowerCase() == 'video/mp2t') {
```
Comment (TheModMaker) [inline lib/hls/hls_parser.js]: There are other `==` checks above that would need to get fixed too.  How about making it lowercase when it is read from the HLS file?

Comment (index23) [reply lib/hls/hls_parser.js]: @TheModMaker I am not sure could this be applicable for all mime type standards. Maybe there are standards with uppercase so this proposal couldn’t fit.

**Comment by joeyparrish** [comment]
It looks like the HLS parser universally converts MIME types to lower case before any of these checks, so this PR is no longer needed.  Thanks!

**Pull request closed without merging**